.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
can you help me to understand why do we need babel, I thought vite can handle all the tasks for us?